src/features/emails/hooks/useEmailStats.ts

```diff
@@ -0,0 +1,29 @@
+import { EmailStats } from '../types';
+import { IFuture } from 'core/caching/futures';
+import { loadItemIfNecessary } from 'core/caching/cacheUtils';
+import { statsLoad, statsLoaded } from '../store';
+import { useApiClient, useAppDispatch, useAppSelector } from 'core/hooks';
+
+interface UseEmailStatsReturn {
+  statsFuture: IFuture<EmailStats>;
+}
+
+export default function useEmailStats(
+  orgId: number,
+  emailId: number
+): UseEmailStatsReturn {
+  const dispatch = useAppDispatch();
+  const apiClient = useApiClient();
+
+  const emailSlice = useAppSelector((state) => state.emails);
+  const statsById = emailSlice.statsById;
+  const statsItem = statsById[emailId];
+
+  const statsFuture = loadItemIfNecessary(statsItem, dispatch, {
+    actionOnLoad: () => statsLoad(emailId),
+    actionOnSuccess: () => statsLoaded({ allTargets: 11, id: 1 }),
```

Comment:
Does this require a new endpoint, or would it make sense to use `/orgs/ID/people/queries/ID/stats` here?

Reply:
Oh didn't know that you can check target number like that! It works actually but email has other stats as well such as 'delivered', 'queue' etc. So I was imagining some endpoint that returns those numbers all together just like call assignment. 
But 'delivered', 'queue' etc belongs to chart stats and it doesn't have priority, so I guess I can use that API and show target numbers instead. Thanks!